features/README.md

```diff
@@ -533,4 +531,4 @@ The PHP Driver follows the architecture of [the C/C++ Driver](http://datastax.gi
 
 ### Persistent sessions
```

Comment:
I feel we can remove this section entirely from the `Architecture` portion as `Persistent Sessions` are discussed at the beginning of this documentation.